src/lib/storage/index/abstract_index.hpp

```diff
@@ -29,11 +29,13 @@ class BaseSegment;
  * As each index has a different way of iterating over its data structures, it has to implement its iterator as well.
  * We might use the impl-pattern similar to the TableScan, but this will be in a future commit.
  *
- * Find more information about this in our wiki: https://github.com/hyrise/hyrise/wiki/BaseIndex and
+ * Find more information about this in our wiki: https://github.com/hyrise/hyrise/wiki/AbstractIndex and
```

Comment:
Not yet. Since the wiki-page references base_index.hpp of the master branch, I would update it after this PR is merged. For now, I just reverted the wiki reference changes.